DESCRIPTION

```diff
@@ -0,0 +1,20 @@
+Package: CFAEpiNow2Pipeline
+Title: EpiNow2 wrapper for deployment to Azure Batch
+Version: 0.0.0.9000
+Authors@R: c(
```

Comment:
Will this get updated as others contribute? Seems like some key players are missing.

Reply:
Yes! I can add names, but wasn't sure how to credit this. Are you thinking everyone who contributed a line of code to pipeline v1 gets an authorship credit here? Something else?

Reply:
Moving to #3